src/components/dialog/dialog.js

```diff
@@ -221,13 +221,14 @@ class Dialog extends Modal {
 
   get closeIcon() {
     if (this.props.showCloseIcon) {
-      return <Icon
-        className="carbon-dialog__close"
+      return (<Icon
```

Comment:
My preference is this. However i understand that it is just my opinion happy for this to go as is

```
return (
  <Icon
    className='carbon-dialog__close'
    data-element='close'
    onClick={ this.props.onCancel }
    type='close'
  />
);
```